tests/run-duration.feature

```diff
@@ -0,0 +1,19 @@
+# This file is part of REANA.
+# Copyright (C) 2023 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+Feature: Run duration
+
+    As a researcher,
```

Comment:
As a researcher, I want to verify that my workflow finishes in a reasonable amount of time, so that I can stay assured that there are no unusual problems with computing resources.